common/rfb/SSecurityTLS.h

```diff
@@ -60,7 +60,6 @@ namespace rfb {
     gnutls_certificate_credentials_t cert_cred;
     char *keyfile, *certfile;
 
-    int type;
```

Comment:
Change looks good, but you can drop the bit about clang in the commit message. We want to keep commit message titles short for formatting reasons.

Reply:
Done.